EIPS/eip-225.md

```diff
@@ -110,27 +111,28 @@ We also define the following per-block constants:
 
  * **`BLOCK_NUMBER`**: Block height in the chain, where the height of the genesis is block `0`.
  * **`SIGNER_COUNT`**: Number of authorized signers valid at a particular instance in the chain.
- * **`SIGNER_INDEX`**: Index of the block signer in the sorted list of current authorized signers.
+ * **`SIGNER_INDEX`**: Zero-based index of the block signer in the sorted list of current authorized signers.
  * **`SIGNER_LIMIT`**: Number of consecutive blocks out of which a signer may only sign one.
    * Must be `floor(SIGNER_COUNT / 2) + 1` to enforce majority consensus on a chain.
 
 We repurpose the `ethash` header fields as follows:
 
- * **`beneficiary`**: Address to propose modifying the list of authorized signers with.
+ * **`beneficiary`** / **`miner`**: Address to propose modifying the list of authorized signers with.
    * Should be filled with zeroes normally, modified only while voting.
    * Arbitrary values are permitted nonetheless (even meaningless ones such as voting out non signers) to avoid extra complexity in implementations around voting mechanics.
    * **Must** be filled with zeroes on checkpoint (i.e. epoch transition) blocks.
-   * Transaction execution **must** use the actual block signer (see `extraData`) for the `COINBASE` opcode.
+   * Transaction execution **must** use the actual block signer (see `extraData`) for the `COINBASE` opcode and transaction fees **must** be attributed to the signer account.
+   * Block execution **should** not distribute any block rewards.
```

Comment:
```suggestion
```
This feels like either a normative change, or an unnecessary change and should be removed either way.  `**should**` recommendations usually don't provide value for standardization because you cannot rely on them.  If they are relied on in some way, then this feels like a normative change which cannot be made to final EIPs.

Reply:
Ok, just added this to help on implementation, but I can follow here along.